web/themes/custom/runroom/templates/layouts/_billboard.html.twig

```diff
@@ -1,15 +1,12 @@
 {% extends '@runroom/layouts/_base.html.twig' %}
 
 {% block main %}
-    {% include '@runroom/components/_skip-link.html.twig' with {
-        target: '#main',
-        title: 'a11y.skip_to.main'|trans
-    } %}
     {% include '@runroom/components/_billboard.html.twig' with {
-        title: block('billboardTitle') is defined ? block('billboardTitle') : '',
-        description: block('billboardDescription') is defined ? block('billboardDescription') : '',
-        class: block('billboardClass') is defined ? block('billboardClass') : '',
-        image: block('billboardImage') is defined ? block('billboardImage') : ''
+        title: block('billboardTitle')|default(''),
+        description: block('billboardDescription')|default(''),
+        class: block('billboardClass')|default(''),
+        image: block('billboardImage')|default('')
     } %}
+
```

Comment:
Aquí yo pasaría el bloque, e iteraria el componente billboard para que tenga `ifs` y defaults a false

Reply:
Y no me quites el bloque de billboard, que es necesario. Pero si que era un error mío haber incluido el billboard en ese bloque. Tendría que ir en el _billboard_

Reply:
los block no hace falta pasarle is defined ni default, porque dentro de ese template ya se controla

Reply:
;)